mead/config/datasets.json

```diff
@@ -143,5 +143,13 @@
     "download": "https://www.dropbox.com/s/lsrdntgu30ftg5l/snips.tar.gz?dl=1",
     "sha1": "c3dd1bff7ad965303065549a05a7f420fc6f3279",
     "label": "snips"
+  },
+  {
+    "label": "conll-multilabel",
```

Comment:
which dataset is this?

Reply:
I took the conll dataset, extracted all the entities for an utterance and the label is the types of the entities, like if an utterance has a 2 locs and a pre the label set would be `PER,LOC`.

It was the easiest dataset I could test all this multilabel stuff on, If you have a better dataset (or we just want to skip having a dataset for it) I can change it

Reply:
This looks like a viable alternative: https://github.com/xiul-msr/e2e_dialog_challenge